src/mapeo-project.js

```diff
@@ -815,7 +832,7 @@ export class MapeoProject extends TypedEmitter {
         relation: [],
       },
     })
-
+    this.#loadingConfig = false
```

Comment:
If anything above `throw`s, we won't reset this boolean. Could we wrap the above in a `try`/`catch`?